pages/components/cards.js

```diff
@@ -0,0 +1,201 @@
+    event.preventDefault();
+    const updatedPosts = posts.map((post) => {
+      if (post.id === postId) {
+        const updatedComments = [...post.comments, post.comment];
+        return { ...post, comments: updatedComments, comment: "" };
+      }
+      return post;
+    });
+    setPosts(updatedPosts);
+  };
+
+  // likestatus Icon toggle
+  const handleLikeClick = (postId) => {
+    const updatedPosts = posts.map((post) => {
+      if (post.id === postId) {
+        return {
+          ...post,
+          likeStatus: !post.likeStatus,
+        };
+      }
+      return post;
+    });
+    setPosts(updatedPosts);
+  };
+
+  // create Click on Image
+  const doubleClickHandler = (postId) => {
+    const updatedPosts = posts.map((post) => {
+      if (post.id === postId) {
+        return { ...post, likePhotos: true, likeStatus: true };
+      }
+      return post;
+    });
+    setPosts(updatedPosts);
+
+    setTimeout(() => {
+      const updatedPosts = posts.map((post) => {
+        if (post.id === postId) {
+          return { ...post, likePhotos: false, likeStatus: true };
+        }
+        return post;
+      });
+      setPosts(updatedPosts);
+    }, 1000);
+  };
+
+  return (
+    <Card>
+      {posts.map((post) => (
+        <div key={post.id}>
+          <PhotoContainer onClick={() => doubleClickHandler(post.id)}>
+            <Photo src={post.Photo} width={100} height={100} alt={post.title} />
+            {post.likePhotos && (
+              <LikeIconImage src="/assets/images/icons/like_fill.png" />
+            )}
+          </PhotoContainer>
+
+          <p>{post.content}</p>
+          <div>
+            <IconContainer>
+              <PhotoIcon
+                src="/assets/images/icons/comment.png"
+                width={30}
+                height={30}
+                alt="comment"
+                onClick={() => handleCommentClick(post.id)}
+              />
+              <PhotoIcon
+                src={
+                  post.likeStatus
+                    ? "/assets/images/icons/like_fill.png"
+                    : "/assets/images/icons/like.png"
+                }
+                width={30}
+                height={30}
+                alt="like"
+                onClick={() => handleLikeClick(post.id)}
+              />
+            </IconContainer>
+            {post.isCommentVisible && (
+              <form onSubmit={(event) => handleCommentSubmit(event, post.id)}>
+                <textarea
+                  placeholder="Enter your comment"
+                  value={post.comment}
+                  onChange={(event) => handleCommentUpdate(event, post.id)}
+                />
+                <button type="submit">Submit</button>
+              </form>
+            )}
+            {post.comments.map((submittedComment, index) => (
+              <CommentText key={index}>{submittedComment}</CommentText>
+            ))}
+          </div>
+          <br />
```

Comment:
I would not suggest to use the `br` element if it's only for distance; please use CSS instead.

Reply:
delete br